tools/nni_cmd/nnictl_utils.py

```diff
@@ -497,7 +497,8 @@ def experiment_clean(args):
         home = str(Path.home())
         local_dir = nni_config.get_config('experimentConfig').get('logDir')
         if not local_dir:
-            local_dir = os.path.join(home, 'nni', 'experiments', experiment_id)
+            local_dir = os.path.join(home, 'nni', 'experiments')
+        local_dir = os.path.join(local_dir, experiment_id)
         local_clean(local_dir)
```

Comment:
we may want to clean `db`, `log`, `trials` directories one by one, to avoid user accidently specify a log dir containing other things while starting experiment.